backend/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM gradle:jdk8 AS build
```

Comment:
What went wrong in this branch? This file is already in main, why should it be added again?

Reply:
The reason for this is rebasing, There are two commits with the same content but different ref hash:
on main:
https://github.com/ehrenamtskarte/ehrenamtskarte/commit/eab148317e5bcee8a37f3744d3768964330695b2

in this pr: https://github.com/ehrenamtskarte/ehrenamtskarte/commit/98be6f07057364ebda70ae903628c5f381bc94d5

Somehow my commits ended up in this branch. This could be because somebody based a branch not on main but on some other branch which was rebased after that.

Conclusion: Don't rebase unless you are sure what you are doing :D

this should not be a problem here because the diffs will merge cleanly if this is merged very soon.

Reply:
Yep, seams I rebased onto the wrong branch by mistake. Even if we should not have had problems merging, I dropped the duplicate commits.